.github/workflows/ci_static-checks.yaml

```diff
@@ -73,6 +73,9 @@ jobs:
             -   name: "Install dependencies"
                 run: composer update --no-interaction --no-scripts
 
+            -   name: "Warmup cache for env dev for PHPStan and Psalm Symfony integrations"
```

Comment:
How about,

> Warmup dev env cache for PHPStan and Psalm Symfony extensions

P.S.
phpstan calls it extension, while psalm - plugin.

Reply:
I've left only `Warmup cache`.